packages/components/src/text-control/style.scss

```diff
@@ -0,0 +1,93 @@
+/** @format */
+
+.muriel-input-text {
+	background: $muriel-white;
+	border: 1px solid $muriel-gray-200;
```

Comment:
> I couldn't find a $muriel-gray-400 - is the "400" here referring to $dark-gray-400?

The `$muriel-gray-` colors are meant to be temporary and are defined to make the newspack components work. See the note here: https://github.com/woocommerce/woocommerce-admin/blob/master/client/stylesheets/abstracts/_variables.scss#L78

We can/should use the `$studio-` variables directly.

Reply:
Thanks for linking to those- got those colors updated 😄